docs/modules/opa/pages/usage-guide/user-info-fetcher.adoc

```diff
@@ -61,91 +61,36 @@ Fetch groups and extra credentials, but not roles.
 
 NOTE: The OAuth2 Client in Keycloak must be given the `view-users` _Service Account Role_ for the realm that the users are in.
 
-// TODO: Document how to use it in OPA regorules, e.g. to authorize based on group membership
-== Example rego rule
-
-[NOTE]
-.User-facing API & API stability
-====
-Since the 24.07 SDP release we provide an example rego rule in our documentation using an HTTP request.
-However, our plan is that the user-facing API of the SPD is *not* the HTTP API of user-info-fetcher, but instead regorules that will automatically be shipped to the OPA server.
-This enables us to make underlying (for example breaking) changes to the HTTP API while keeping the rego rules API stable.
-
-The documentation will be updated to use the deployed rego rules once available.
-====
-
-[NOTE]
-.About unencrypted HTTP
-====
-The User info fetcher serves endpoints over clear-text HTTP.
-
-It is intended to only be accessed from the OPA Server via _localhost_ and to not be exposed outside of the Pod.
-====
-
-[source,rego]
-----
-package test # <1>
-
-# Define a function to lookup by username
-userInfoByUsername(username) := http.send({
-  "method": "POST",
-  "url": "http://127.0.0.1:9476/user",
-  "body": {"username": username}, <2>
-  "headers": {"Content-Type": "application/json"},
-  "raise_error": true
-}).body
-
-# Define a function to lookup by a stable identifier
-userInfoById(id) := http.send({
-  "method": "POST",
-  "url": "http://127.0.0.1:9476/user",
-  "body": {"id": id}, <3>
-  "headers": {"Content-Type": "application/json"},
-  "raise_error": true
-}).body
-
-currentUserInfoByUsername := userInfoByUsername(input.username)
-currentUserInfoById := userInfoById(input.id)
-----
-
-<1> The package name is important in the OPA URL used by the product.
-<2> Lookup by username
-<3> Lookup by id
-
-For more information on the request and response payloads, see <<_user_info_fetcher_api>>
-
 == User info fetcher API
 
-HTTP Post Requests must be sent to the `/user` endpoint with the following header set: `Content-Type: application/json`.
+User information can be retrieved from regorules using the functions `userInfoByUsername(username)` and `userInfoById(id)` in `data.stackable.opa.userinfo.v1`.
 
-You can either lookup the user info by stable identifier:
+An example of the returned structure:
 
 [source,json]
 ----
 {
   "id": "af07f12c-a2db-40a7-93e0-874537bdf3f5",
+  "username": "alice",
+  "groups": [
+    "/superset-admin"
+  ],
+  "customAttributes": {}
 }
 ----
 
-or by the username:
+For example, the following rule will allow access for users in the `/admin` group:
```

Comment:
**question**: Do we want to use the same group throughout the docs (ie: settle on `/superset-admin` or `/admin`)?

Reply:
Good catch, b8a9b0add6eedb3e1c8e316568f93065f4e6aeb9